sdk/servicebus/service-bus/samples/browserSample.html

```diff
@@ -9,16 +9,16 @@
 <script src="../browser/service-bus.js"></script>
 
 <script>
-  // SBC holds reference to the ServiceBusClient global object exposed by the imported script
-  const SBC = Azure.Messaging.ServiceBus.ServiceBusClient;
+  // sbClient holds reference to the ServiceBusClient global object exposed by the imported script
```

Comment:
`SBC` should be `ServiceBusClient` and `serviceBusClientObj` should be `sbClient`